docs/ja/docs/advanced/custom-response.md

```diff
@@ -0,0 +1,223 @@
+# カスタムレスポンス - HTML、ストリーム、ファイル、その他のレスポンス
+
+デフォルトでは、**FastAPI** は `JSONResponse` を使ってレスポンスを返します。
+
+[レスポンスを直接返す](response-directly.md){.internal-link target=_blank}で見たように、 `Response` を直接返すことでこの挙動をオーバーライドできます。
+
+しかし、`Response` を直接返すと、データは自動的に変換されず、ドキュメントも自動生成されません (例えば、生成されるOpenAPIの一部としてHTTPヘッダー `Content-Type` に特定の「メディアタイプ」を含めるなど) 。
+
+しかし、*path operationデコレータ* に、使いたい `Response` を宣言することもできます。
+
+*path operation関数* から返されるコンテンツは、その `Response` に含まれます。
+
+そしてもし、`Response` が、`JSONResponse` や `UJSONResponse` の場合のようにJSONメディアタイプ (`application/json`) ならば、データは *path operationデコレータ* に宣言したPydantic `response_model` により自動的に変換 (もしくはフィルタ) されます。
+
+!!! note "備考"
+    メディアタイプを指定せずにレスポンスクラスを利用すると、FastAPIは何もコンテンツがないことを期待するので、生成されるOpenAPIドキュメントにレスポンスフォーマットが記載されません。
+
+## `ORJSONResponse` を使う
+
+例えば、パフォーマンスを出したい場合は、<a href="https://github.com/ijl/orjson" class="external-link" target="_blank">`orjson`</a>をインストールし、`ORJSONResponse`をレスポンスとしてセットすることができます。
+
+使いたい `Response` クラス (サブクラス) をインポートし、 *path operationデコレータ* に宣言します。
+
+```Python hl_lines="2  7"
+{!../../../docs_src/custom_response/tutorial001b.py!}
+```
+
+!!! info "情報"
+    パラメータ `response_class` は、レスポンスの「メディアタイプ」を定義するために利用することもできます。
+
+    この場合、HTTPヘッダー `Content-Type` には `application/json` がセットされます。
+
+    そして、OpenAPIにはそのようにドキュメントされます。
+
+!!! tip "豆知識"
+    `ORJSONResponse` は、現在はFastAPIのみで利用可能で、Starletteでは利用できません。
+
+## HTMLレスポンス
+
+**FastAPI** からHTMLを直接返す場合は、`HTMLResponse` を使います。
+
+* `HTMLResponse` をインポートする。
+* *path operation* のパラメータ `content_type` に `HTMLResponse` を渡す。
+
+```Python hl_lines="2  7"
+{!../../../docs_src/custom_response/tutorial002.py!}
+```
+
+!!! info "情報"
+    パラメータ `response_class` は、レスポンスの「メディアタイプ」を定義するために利用することもできます。
```

Comment:
```suggestion
    パラメータ `response_class` は、レスポンスの「メディアタイプ」を定義するために利用されます。
```

More simple. And the original sentence doesn't seem to emphasize that you can do that much.